src/test/resources/com/browserstack/user.feature

```diff
@@ -0,0 +1,12 @@
+Feature: User Feature
+
+  Scenario: Login as User with no image loaded
+    Given I navigate to website
+    And I SignIn as "image_not_loading_user" with "testingisfun99" password
+    Then I should see no image loaded
```

Comment:
This scenario needs to be failed on purpose using assertions and marked as "failed" on BrowserStack.